### PR TITLE
Move some network configuration into the plugin

### PR DESCRIPTION
**Pull request by ejj**
This series moves OVSDB port creation, and some OpenFlow setup into the plugin.  Once this series is in, containers will be able to resolve their dependencies the first time they try.

### CODE REVIEW

File: api/client/mocks/Getter.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package mocks
 
-import "github.com/NetSys/quilt/api/client"
-import "github.com/stretchr/testify/mock"
+import client "github.com/NetSys/quilt/api/client"
+import mock "github.com/stretchr/testify/mock"
```
Comment (PenguinToast) [inline api/client/mocks/Getter.go]: Is there any reason we name the imports here?

File: minion/network/openflow/openflow.go [hunk 2]
```diff
@@ -126,10 +127,17 @@ Table_2 {
...
-	VethPort  int
-	Mac       string
+// A Container that needs OpenFlow rules installed for it.
+type Container struct {
+	Veth  string
+	Patch string
+	Mac   string
+}
+
+type container struct {
```
Comment (kklin) [inline minion/network/openflow/openflow.go]: I find the fact that there's an exported and unexported `container` with the same fields somewhat confusing. Maybe we could tweak the name?

File: api/client/mocks/Getter.go (hunk 1 above)
Comment (ejj) [inline api/client/mocks/Getter.go]: Ah this is auto-generated code.  Apparently the library that generated it changed.

File: minion/network/openflow/openflow.go (hunk 2 above)
Comment (ejj) [inline minion/network/openflow/openflow.go]: Meh, I actually like the name.  It's literally the internal version of a container.

**Pull request merged**